### DISCUSSION TITLE
`.publisherScope` deprecation.

### DISCUSSION THREAD

**Discussion by OguzYuuksel** [pointfreeco/swift-composable-architecture#2133] [Beta]
Hello, I have seen the warnings that this method is deprecated and wanted to open this topic as stated in the warning.

How we use it like we have two different Redux architectures in our codebase, and we are moving our codebase to the TCA, so we had to provide a bridge layer between two Stores and we are using this method to sync store states properly and listen to action changes.

**Comment by stephencelis** [comment]
@OguzYuuksel Can you share some code that shows how you use `publisherScope`? We're wondering if the same can be accomplished with other tools.
